PortfolioApp/Users/UserComposer.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+final class UserComposer {
```

Comment:
Add private init() {} classes, structs with only static variables, or functions should have a private init. They can't and won't be initialized